python/plugins/MetaSearch/ui/newconnectiondialog.ui

```diff
@@ -68,7 +68,11 @@
        </widget>
       </item>
       <item row="2" column="1">
-       <widget class="QLineEdit" name="lePassword"/>
+       <widget class="QgsPasswordLineEdit" name="lePassword">
+        <property name="echoMode">
+         <enum>QLineEdit::Password</enum>
```

Comment:
Nitpicking, but the QgsPasswordLineEdit will do it for you :)
https://github.com/qgis/QGIS/blob/master/src/gui/qgspasswordlineedit.cpp#L54